hopsworks-common/src/main/java/io/hops/hopsworks/common/project/ProjectController.java

```diff
@@ -724,6 +731,17 @@ public Project findProjectById(Integer id) throws ProjectException {
           addServiceDataset(project, user, Settings.ServiceDataset.EXPERIMENTS, dfso, udfso);
           addKibana(project);
         }
+        break;
+      case FEATURESTORE:
+        addServiceFeaturestore(project, user, dfso);
+        addServiceDataset(project, user, Settings.ServiceDataset.TRAININGDATASETS, dfso, udfso);
+        //Enable Jobs service at the same time as featurestore
+        if (!projectServicesFacade.isServiceEnabledForProject(project, ProjectServiceEnum.JOBS)) {
+          if (!projectServicesFacade.isServiceEnabledForProject(project, ProjectServiceEnum.JUPYTER)) {
+            addServiceDataset(project, user, Settings.ServiceDataset.EXPERIMENTS, dfso, udfso);
+            addKibana(project);
+          }
+        }
```

Comment:
@Limmen add break. Although it's the last case, it's good to add it because if we add more cases in the future chances are we will forget to add a break;

Reply:
Fixed